manifests/addons/values-kiali.yaml

```diff
@@ -0,0 +1,11 @@
+# Create a Kiali CR in the istio-system namespace, which is where Kiali will be installed.
+# We set up anonymous authentication as this is for demos.
+cr:
```

Comment:
nit: i wonder if we should name this something more like `kiali` or `instance` or `resource`.

Reply:
that's actually the setting in the new helm chart: https://github.com/kiali/kiali-operator/pull/93/files#diff-26f2c186ac99e122d20cefaa0a5cd660R43

We could change it if we can think of a better name. Just "kiali" is no good (not really descriptive of what it is). Perhaps `kialiInstance` or `kialiCR` is better (we call it `cr` today because it is the config for the "Kiali CR" object that is to be created - we call this thing "Kiali CR" all throughout the docs). If we change it, it will require changes throughout that kiali PR though. Would take some time to go through it all and rename it. I am hoping to get this Kiali Helm Chart merged within the next day or two.